### DISCUSSION TITLE
Is windows 11 supported?

### DISCUSSION THREAD

**Discussion by zapta** [hbldh/bleak#1054] [Q&A]
I have a bleak program that runs on windows 10 and fails on windows 11 (it can find the device by address but does not connect).

 Is windows 11 supported?  I am not sure how to parse this statement from the bleak page.

"Supports Windows 10, version 16299 (Fall Creators Update) or greater"

**Comment by dlech** [comment]
Yes, Windows 11 is "greater" than 10.

**Comment by zapta** [comment]
Thanks. That's good.

The text is ambiguous, can also be parsed as:

"Supports Windows 10, {version 16299 (Fall Creators Update) or greater}

**Comment by LakshaySahni** [comment]
@zapta is your code working now? I'm facing the same issue where my bleak program runs perfectly on windows 10. but not on windows 11. Any idea how to solve this?

**Reply by zapta** [reply]
The issue continues here with more information https://github.com/hbldh/bleak/issues/1132 . My code works after I removed the import of pkg_resources in my program.

**Reply by LakshaySahni** [reply]
Got it. Have been following #1132 too, but I'm not importing pkg_resources and am getting the following error in trace:

`UnicodeEncodeError: 'charmap' codec can't encode characters in position 79-91: character maps to <undefined>`

after I run:
`await client.connect()`

Any idea on how to resolve this would be helpful! thanks :)

**Reply by dlech** [reply]
> `UnicodeEncodeError: 'charmap' codec can't encode characters in position 79-91: character maps to <undefined>`

possibly fixed by https://github.com/hbldh/bleak/commit/b0074d3d12463ac5e56899d024ece9b63a48c621
